src/index.js

```diff
@@ -352,7 +352,15 @@ const replaceInWindows = (
   )
 }
 
-const expandText = (text) => {
+const expandText = (inputText) => {
+  // Handle formatting bold, italics, and strikethrough within single and double quotes
+  const text = inputText.replace(/(['"])(?:(?=(\\?))\2.)*?\1/g, (quotedSegment) => {
+    return quotedSegment
+      .replace(/\*([^*]+)\*/g, `${boldOpeningPatternString}$1${boldClosingPatternString}`)
+      .replace(/_([^_]+)_/g, `${italicOpeningPatternString}$1${italicClosingPatternString}`)
+      .replace(/~([^~]+)~/g, `${strikethroughOpeningPatternString}$1${strikethroughClosingPatternString}`)
+  })
```

Comment:
This would fail for cases where we have spaces inside the quotes, something like:
`"TEST_MESSAGE_TEST  _text_"`
This would convert the `MESSAGE` into italics as well. 
I was thinking maybe when we enable space padding, the place where we are checking not to format in case of spaces, we could also add the check for quotes?